src/olympia/accounts/tasks.py

```diff
@@ -14,6 +16,7 @@ def primary_email_change_event(email, uid, timestamp):
     """Process the primaryEmailChangedEvent."""
     try:
         profile = UserProfile.objects.get(fxa_id=uid)
+        timestamp = dateutil_parser(timestamp)
```

Comment:
We could just change `process_fxa_event` to not do `datetime.fromtimestamp` first and pass along the raw int value to the task, then convert it from a unix-style timestamp in the task instead.  Seems superfluous to do int>datetime>string>datetime

Reply:
`event.get('ts', '')` should be `event.get('ts', 0)` :/

Reply:
https://github.com/mozilla/addons-server/pull/8383/commits/91d1b68f0df760cbb644e0e671b2b3d8f9caae11